Makefile

```diff
@@ -1,39 +1,47 @@
-# Package configuration
-PROJECT = sourced-ui
-
-# superset upstream configuration
+# Superset upstream configuration
 SUPERSET_REPO = https://github.com/apache/incubator-superset.git
 SUPERSET_VERSION = 0.33.0rc1
 SUPERSET_REMOTE = superset
-# directory to sync superset upstream with
+# Directory to sync superset upstream with
 SUPERSET_DIR = superset
-# directory with custom code to copy into SUPERSET_DIR
+# Directory with custom code to copy into SUPERSET_DIR
 PATCH_SOURCE_DIR = srcd
-# name of the superset docker image to build
-SUPERSET_IMAGE_NAME ?= srcd/sourced-ui
 
-# Including ci Makefile
-CI_REPOSITORY ?= https://github.com/src-d/ci.git
-CI_PATH ?= $(shell pwd)/.ci
-CI_VERSION ?= v1
+# Name of the docker image to build
+DOCKER_IMAGE_NAME ?= srcd/sourced-ui
+# Docker registry where the docker image should be pushed to.
+DOCKER_REGISTRY ?= docker.io
+# Docker organization to be used at the docker image name.
+DOCKER_ORG ?= srcd
+# Username used to login on the docker registry.
+DOCKER_USERNAME ?=
+# Password used to login on the docker registry.
+DOCKER_PASSWORD ?=
+# When `make docker-push`, setting DOCKER_PUSH_LATEST to any non-empty value
+# will cause make docker-push to also push the latest tag.
+DOCKER_PUSH_LATEST ?=
 
-MAKEFILE := $(CI_PATH)/Makefile.main
-$(MAKEFILE):
-	git clone --quiet --branch $(CI_VERSION) --depth 1 $(CI_REPOSITORY) $(CI_PATH);
+# Build information
+VERSION ?= latest
+# Travis CI
+ifneq ($(TRAVIS_TAG), )
```

Comment:
Already in [src-d/ci/Makefile#L58](https://github.com/src-d/ci/blob/v1/Makefile.main#L58)